crates/cli/src/main.rs

```diff
@@ -4,8 +4,25 @@ use yansi::Paint as _;
 
 #[tokio::main]
 async fn main() {
+    banner();
     let args = Args::parse();
     if let Err(err) = run(args.command).await {
         eprintln!("{}", err.to_string().red())
     }
 }
+
+// Banner edittable 
+fn banner() {
+    println!(
+        "{}",
+        "
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    ╔═╗╔═╗╦  ╔╦╗╔═╗╔═╗╦═╗     Solidity Package Manager 
+    ╚═╗║ ║║   ║║║╣ ║╣ ╠╦╝          built in rust
+    ╚═╝╚═╝╩═╝═╩╝╚═╝╚═╝╩╚═    github.com/mario-eth/soldeer   
+         soldeer.xyz              x.com/m4rio_eth
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+"
+        .bright_cyan()
+    );
+}
```

Comment:
```suggestion
/// Generate and print a banner
fn banner() {
    println!(
        "{}",
        format!(
            "
+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
    ╔═╗╔═╗╦  ╔╦╗╔═╗╔═╗╦═╗       Solidity Package Manager
    ╚═╗║ ║║   ║║║╣ ║╣ ╠╦╝
    ╚═╝╚═╝╩═╝═╩╝╚═╝╚═╝╩╚═     github.com/mario-eth/soldeer
           v{}                       soldeer.xyz
+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
",
            env!("CARGO_PKG_VERSION")
        )
        .bright_cyan()
    );
}
```

Reply:
OK looks like u already did the changes , I received a notification , is there some action i need to do ? im not exactly a hardcore dev